recipe/build.sh

```diff
@@ -1,16 +1,10 @@
 #!/bin/bash
 
-# FIXME: This is a hack to make sure the environment is activated.
-# The reason this is required is due to the conda-build issue
-# mentioned below.
-#
-# https://github.com/conda/conda-build/issues/910
-#
-source activate "${CONDA_DEFAULT_ENV}"
+# rm -rf $PREFIX/lib/*.la
```

Comment:
@gillins and @kmuehlbauer this is a radical solution, but it saves us from the bad libtool archive files in `defaults`. The alternative is to pin `libxml2 to 2.6.3` to ensure that only the conda-forge version will be installed.